src/Components/Landing/Landing.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-absolute-path */
+import React from "react";
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Don't bypass eslint rules writing comments. Try to search in Google for the solution why there is error and fix it.

Also no need to import React after version 17 of React because of the new JSX Transformed introduced.

[Reference](https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj)